### PR TITLE
Track/update : update tracking in web manager

### PR DESCRIPTION
**Pull request by whispyy**

### CODE REVIEW

File: client/app/domain/anycast/domain-dns-anycast.controller.js [hunk 1]
```diff
@@ -55,6 +57,13 @@ angular.module("App").controller(
 
         displayBC () {
             this.$scope.resetAction();
+            this.atInternet.trackOrder({
+                name: `[domain]${this.optionName}`,
+                page: "web::domain::product",
+                orderId: this.order.orderId,
```
Comment (JDownloader) [inline client/app/domain/anycast/domain-dns-anycast.controller.js]: Those 3 lines are the same across the controllers, may be proxying them in a new service could be practical for future requests.
```
orderId: this.order.orderId,
priceTaxFree: this.order.prices.withoutTax.value,
status: 1
```

File: client/app/email-domain/order/email-domain-order.controller.js [hunk 2]
```diff
@@ -129,6 +131,13 @@ angular.module("App").controller(
...
 
         openBc () {
+            this.atInternet.trackOrder({
+                name: `[MXPlan]MX-${this.selectedOrder.offer.offer}`,
+                page: "web::mx-plan",
+                orderId: this.order.orderId,
+                priceTaxFree: this.order.prices.withoutTax.value,
+                status: 1
+            });
             window.open(this.order.url);
```
Comment (JDownloader) [inline client/app/email-domain/order/email-domain-order.controller.js]: same

File: client/app/domain/anycast/domain-dns-anycast.controller.js [hunk 3]
```diff
@@ -55,6 +57,13 @@ angular.module("App").controller(
...
+                priceTaxFree: this.order.prices.withoutTax.value,
+                status: 1
+            });
             this.Alerter.success(this.$scope.tr("domain_order_dns_anycast_success", [this.url]), this.$scope.alerts.main);
             window.open(this.url, "_blank");
```
Comment (JDownloader) [inline client/app/domain/anycast/domain-dns-anycast.controller.js]: It's not your, using `$window.open(...)` give more flexibility

File: client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js [hunk 4]
```diff
@@ -179,6 +180,12 @@ angular.module("App").controller(
...
                     }
+                    this.atInternet.trackOrder({
+                        name: `[hosting]${this.model.offer}`,
+                        page: "web::domain::product",
+                        orderId: order.orderId,
+                        priceTaxFree: order.prices.withoutTax.value
+                    });
 
                     this.Alerter.success(this.$scope.tr("domain_order_hosting_finish_success", [order.url]), this.$scope.alerts.main);
                     window.open(order.url, "_blank");
```
Comment (JDownloader) [inline client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: same

File: client/app/hosting/multisite/order/hosting-multisite-order.controller.js [hunk 5]
```diff
@@ -22,6 +23,10 @@ angular.module("App").controller(
         }
 
         orderDomain () {
+            this.atInternet.trackEvent({
+                event: "hosting-add-domain",
+                page: "web::hosting"
+            });
             switch (this.selected.action) {
             case this.model.actions.ORDER:
                 this.$window.open(this.$scope.urlDomainOrder);
```
Comment (JDownloader) [inline client/app/hosting/multisite/order/hosting-multisite-order.controller.js]: :-)

File: client/app/hosting/offer/upgrade/hosting-offer-upgrade.controller.js [hunk 6]
```diff
@@ -102,6 +103,13 @@ angular.module("App").controller(
...
                 .then((order) => {
                     this.Alerter.success(this.$scope.tr("hosting_order_upgrade_success", [order.url, order.orderId]), this.$scope.alerts.main);
+                    this.atInternet.trackOrder({
+                        name: `[hosting]${this.model.capacity}`,
+                        page: "web::hosting",
+                        orderId: order.orderId,
+                        priceTaxFree: order.prices.withoutTax.value,
+                        status: 1
+                    });
                     window.open(order.url, "_blank");
```
Comment (JDownloader) [inline client/app/hosting/offer/upgrade/hosting-offer-upgrade.controller.js]: same

File: client/app/private-database/order/sql-database-order-controller.js [hunk 7]
```diff
@@ -384,6 +385,13 @@ angular.module("App").controller(
...
 
         openBc () {
+            this.atInternet.trackOrder({
+                name: `[SQL-${this.model.type}]${this.model.version || this.model.dbPack}`,
+                page: "web::sql-order",
+                orderId: this.order.orderId,
+                priceTaxFree: this.order.prices.withoutTax.value,
+                status: 1
+            });
             window.open(this.order.url);
```
Comment (JDownloader) [inline client/app/private-database/order/sql-database-order-controller.js]: same

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 8]
```diff
@@ -94,7 +94,10 @@ <h3 class="oui-tile__title" data-i18n-static="domain_dashboard_general_informati
                                             <button class="btn btn-default ml-auto my-auto text-wrap" type="button"
                                                     data-i18n-static="domain_configuration_dnsanycast_title_button"
                                                     data-ng-click="setAction('anycast/domain-dns-anycast')"
-                                                    data-ng-if="ctrlTabGeneralInformations.domain.hasDnsAnycast === false">
+                                                    data-ng-if="ctrlTabGeneralInformations.domain.hasDnsAnycast === false"
+                                                    data-track-on="click"
+                                                    data-track-name="web::domain::genetal-informations::activation-dns-anycast"
```
Comment (cbourgois) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: `general` maybe ?

Comment (whispyy) [reply client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: THANKS :)

**Pull request merged**